src/components/shared/Entity/IncompatibleCollections/index.tsx

```diff
@@ -37,8 +39,9 @@ function IncompatibleCollections() {
                         </Typography>
                     </Box>
 
-                    <CollectionsList />
-
+                    <Box sx={{ maxHeight: 200, overflow: 'auto' }}>
+                        <CollectionsList />
+                    </Box>
```

Comment:
@psFried - Added to make the list scroll when it got too long. I know we do not do this up above with errors but with the two sections sometimes displaying at the same time (at least when I tested) the page was getting a little ridiculously big. Really just a personal tweak.